index.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express')
+const movies = require('./movies')
+const { findTitle, findDirector } = require('./controllers/movies')
+
+const app = express()
+
+app.get('/movies', (request, response) => {
+  return response.send( { movies } );
+})
+
+app.get('/movies/:title', findTitle)
+
+app.get('/movies/director/:director', findDirector)
```

Comment:
I recommend switching this from '/movies/director/:director' to simply '/movies/:director', similar to how you have '/movies/:title' set up.